commit 2a55cc89030dc0c7b488a515147cab4aac6041b0 (refs/changes/11/3311/3)
Author: David Pacheco <dap@joyent.com>
Date:   2018-02-08T12:05:24-08:00 (1 year, 8 months ago)
    
    MORAY-455 moray asks fast to hold onto too many recent RPC requests
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 5f3d068..1801bb9 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.4.2
+
+* [MORAY-455](http://smartos.org/bugview/MORAY-455) moray asks fast to hold onto too many recent RPC requests
+
 ## v3.4.1
 
 * [MORAY-422](http://smartos.org/bugview/MORAY-422) node-moray could smooth over connection shuffling
diff --git a/lib/client.js b/lib/client.js
index 50bab99..1a2a6da 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -35,7 +35,7 @@ var parseMorayParameters = require('./client_params').parseMorayParameters;
 
 ///--- Default values for function arguments
 
-var fastNRecentRequests         = 30;
+var fastNRecentRequests         = 5;
 var dflClientTcpKeepAliveIdle   = 10000;  /* milliseconds */
 
 /*
diff --git a/package.json b/package.json
index a310842..4f4282c 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "moray",
     "description": "Moray client library",
-    "version": "3.4.1",
+    "version": "3.4.2",
     "author": "Joyent (joyent.com)",
     "keywords": [ "moray" ],
     "main": "./lib/index.js",
